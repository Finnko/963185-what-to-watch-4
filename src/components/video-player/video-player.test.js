import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

const mock = {
  isPlaying: true,
  videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  posterSrc: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`
};

describe(`VideoPlayer snapshot tests`, () => {
  it(`VideoPlayer should render correctly`, () => {
    const {isPlaying, videoSrc, posterSrc} = mock;
    const tree = renderer
      .create(
          <VideoPlayer
            isPlaying={isPlaying}
            videoSrc={videoSrc}
            posterSrc={posterSrc}
          />,
          {
            createNodeMock: () => {
              return {};
            }
          }
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
