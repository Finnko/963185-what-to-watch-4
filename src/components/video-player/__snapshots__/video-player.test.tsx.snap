// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer snapshot test VideoPlayer should render correctly 1`] = `
<video
  height="175"
  poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  width="280"
/>
`;
